pkg/daemon/ceph/client/osd.go

```diff
@@ -180,6 +180,26 @@ func EnableScrubbing(context *clusterd.Context, clusterName string) (string, err
 	return string(buf), nil
 }
 
+// SetNoOut execs `ceph osd set noout`
+func SetNoOut(context *clusterd.Context, clusterName string) (string, error) {
+	args := []string{"osd", "set", "noout"}
+	buf, err := ExecuteCephCommand(context, clusterName, args)
```

Comment:
If there is a ceph command to detect if `noout` is already set, you could also call `ExecuteCephCommandDebugLog()` instead of `ExecuteCephCommand()` to cut down on the logging.

Reply:
The only ceph command I could find involved parsing the status message. @leseb is there another way?

Reply:
You can run `ceph osd dump -f json` and parse the `flags_set` key which contains a list of enabled flags, see:

```
    "flags_set": [
        "noout",
        "pglog_hardlimit",
        "purged_snapdirs",
        "recovery_deletes",
        "sortbitwise"
    ],
```

Reply:
Thanks, incorporated it.